src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1530,10 +1530,47 @@ def receive_timeseries_from_csv(
         compute_timeseries_properties(dict_asset)
 
 
+def replace_nans_in_timeseries_with_0(timeseries, label):
+    """
+
+    Replaces nans in the timeseries (if any) with 0
+
+    Parameters
+
+    ----------
+    timeseries: pd.Series
+        timeseries in dict_asset (having nan value(s) if any), also of parameters
```

Comment:
what does "also of parameters" means here?

Reply:
parameters can be defined as scalars and as timeseries, and the function is not only applied to resource and demand data, but also those timeseries.